src/error.rs

```diff
@@ -1583,3 +1595,37 @@ impl ToDiagnostic<FileId> for ReplError {
         }
     }
 }
+
+impl std::fmt::Display for RustDeserializationError {
```

Comment:
I wonder if this error shouldn't stay in `deserializer`. There is no strong rationale (unfortunately) with respect to what error should go in `error` and what can stay in the module itself, but I would say that `error.rs` is concerned with top-level errors that the user may face directly (evaluation, IO, typechecking, etc.). For example I think they all implement the `ToDiagnostic` trait to print them nicely using codespan.

This is not the case of `RustDeserializationError`, which seems rather to be directed toward crates consuming the deserializer implementation.